### ISSUE TITLE
403 error for deleting annotations

### ISSUE THREAD

**Issue by gigamorph** [yale-web-technologies/mirador-annotations#143]
If an annotation is orphaned -- the annotation that this annotation targets has been deleted -- then the current version of auth check is bound to fail because it cannot get the canvas ID to determine the user's permission.

**Comment by hshyk** [comment]
Should there be a cron job or logic to cascade the delete for orphaned annotations?

**Comment by gigamorph** [comment]
Yes, maybe we should delete all "dependent" annotations too when we delete the "parent" annotation. Then I'll have to put a warning that lists the annotations that will be deleted.
